hooks/use-autoclaim.ts

```diff
@@ -44,7 +44,7 @@ function useAutoclaim(contractConfig: ContractNetwork | undefined, address: `0x$
   const register = useCallback(
     async (days: number, amount: number) => {
       if (contractConfig) {
-        const timeStamp = BigInt(days * 86400000);
+        const timeStamp = BigInt(days * 86400);
```

Comment:
Better to move magic number to constant with name as description of what this number is about.

Reply:
Fixed thanks :)